### PR TITLE
return 'SuccessfulRescale' only

### PR DESCRIPTION
**Pull request by GLYASAI**
return 'SuccessfulRescale' only, not 'FailedGetResourceMetric' or other err

### CODE REVIEW

File: db/mysql/dao/tenants.go [hunk 1]
```diff
@@ -1807,7 +1807,8 @@ func (t *TenantServiceScalingRecordsDaoImpl) UpdateOrCreate(new *model.TenantSer
 // ListByServiceID -
 func (t *TenantServiceScalingRecordsDaoImpl) ListByServiceID(serviceID string, offset, limit int) ([]*model.TenantServiceScalingRecords, error) {
 	var records []*model.TenantServiceScalingRecords
-	if err := t.DB.Where("service_id=?", serviceID).Offset(offset).Limit(limit).Order("last_time desc").Find(&records).Error; err != nil {
+	if err := t.DB.Where("service_id=? and reason=?", serviceID, "SuccessfulRescale").
```
Comment (barnettZQG) [inline db/mysql/dao/tenants.go:1810]: Should it not be saved, or not queried?

Comment (GLYASAI) [reply db/mysql/dao/tenants.go:1810]: ok

File: worker/appm/store/store.go [hunk 2]
```diff
@@ -1183,6 +1183,10 @@ func (a *appRuntimeStore) evtEventHandler() cache.ResourceEventHandlerFuncs {
 	return cache.ResourceEventHandlerFuncs{
 		AddFunc: func(obj interface{}) {
 			evt := obj.(*corev1.Event)
+			if evt.Reason != "SuccessfulRescale" {
+				return
```
Comment (barnettZQG) [inline worker/appm/store/store.go:1187]: It is recommended to print the log.

Comment (GLYASAI) [reply worker/appm/store/store.go:1187]: UpdateFunc will be triggered continuously, and there will be many logs. You can use kubectl to view hpa events.

**Pull request closed without merging**